src/sinks/file/mod.rs

```diff
@@ -143,6 +143,10 @@ impl SinkConfig for FileSinkConfig {
     fn sink_type(&self) -> &'static str {
         "file"
     }
+
+    fn can_acknowledge(&self) -> bool {
+        true
```

Comment:
Here I'm wondering about `fsync`. I guess the ack here just indicates that we called `write` successfully? We may want to document the guarantees of this one explicitly.

Reply:
It does handle write errors and the like, which is what the status is based on, but you're right, it doesn't `fsync` or `fdatasync` the file.

Reply:
Added issue #11473

Reply:
Done in 4af2d907f